src/main/scala/wdltool/graph/WomGraph.scala

```diff
@@ -117,12 +119,49 @@ class WomGraph(graphName: String, graph: Graph) {
          """.stripMargin
 
       val scatterVariableInputLink = s"$scatterVariableExpressionId -> ${scatter.scatterVariableMapping.graphInputNode.singleOutputPort.graphId} [style=dashed ltail=$scatterVariableClusterName arrowhead=none]"
-      val outputLinks = scatter.outputMapping map { outputPort => s"${outputPort.outputToGather.singleInputPort.graphId} -> ${outputPort.graphId} [style=dashed arrowhead=none]" }
-      val links: Set[String] = outputLinks + scatterVariableInputLink
 
-      NodesAndLinks(Set(node), links)
+      NodesAndLinks(Set(node), Set(scatterVariableInputLink))
     }
-    innerGraph |+| scatterVariableNodesAndLinks
+    val outputLinks = scatter.outputMapping map { outputPort => s"${outputPort.outputToGather.singleInputPort.graphId} -> ${outputPort.graphId} [style=dashed arrowhead=none]" }
+    (innerGraph |+| scatterVariableNodesAndLinks).withLinks(outputLinks)
+  }
+
+  def internalConditionalNodesAndLinks(conditional: ConditionalNode): NodesAndLinks = {
+    val innerGraph = listAllGraphNodes(conditional.innerGraph) wrapNodes { n =>
+      s"""
+         |subgraph $nextCluster {
+         |  style=filled;
+         |  fillcolor=lightgray;
+         |${indentAndCombine(n)}
+         |}
+         |""".stripMargin
+    }
+    val conditionNodesAndLinks = {
```

Comment:
conditional or scatter, not sure what's going on here

Reply:
I added a comment and removed the `scatter` references inside 😳